charts/k6-operator/values.yaml

```diff
@@ -39,6 +39,8 @@ authProxy:
   readinessProbe: {}
 
 manager:
+  # manager.replicaCount -- number of controller-manager replicas to run (default: 1)
+  replicaCount: 1
```

Comment:
I've just now noticed this: can we go with `replicas` here? It's under `manager` so should be clear what it is and it's what's used in Kubernetes definition.

Reply:
Got it, done

Reply:
Sounds good, done.